### DISCUSSION TITLE
[Feature] Drop files without displaying album content

### DISCUSSION THREAD

**Discussion by Extarys** [immich-app/immich#6230] [Feature Request]
### The feature

I would like to allow users to upload photos without being able to access the album's content.

I'm currently trying to gather pictures from people that were at an event and then send them a link to download everything once everyone uploaded all their pictures.

But many people aren't tech-savvy and they get confused.

Making a big "Drop files" button or something with only the album title/description would be more user-friendly for them.

Note: I tried Immich for the first time today and so far I enjoy it a lot.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by DomJeal** [comment]
I'm in the same boat, I actually thought the current album page would be fine as-is, but I've just sent it to my fiancée who, admittedly, isn't very tech-savvy and the first thing she asked was "how do I upload?".

I may add a quick instruction to the QR/link printouts I plan to do, but a very simple "idiotproof" upload only page would be nice!

**Comment by bo0tzz** [comment]
#5681

**Reply by Extarys** [reply]
Thanks, for sake of clarity, I posted this link to the older conversation.
